bucket-policies/rubin-summit-users-policy.json

```diff
@@ -0,0 +1,20 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [{
+    "Sid": "allow-read-write-rubin-summit-users",
+    "Effect": "Allow",
+    "Principal": {
+      "AWS": [
+        "arn:aws:iam:::user/prompt-processing-dev"
+      ]
+    },
+    "Action": [
+      "s3:*"
```

Comment:
This is more than should strictly be necessary; restricting to object manipulations would be sufficient.